### PR TITLE
Activate Services in Terraform Instead of in Forseti

### PR DESCRIPTION
**Pull request by johnrevans6**
Forseti has several service dependencies that should be managed by Terraform. Pulled these dependencies in to the module. Note, calling destroy on the Compute Engine API when there are still resources in use will cause terraform to fail to deactivate the service.

### CODE REVIEW

File: examples/simple/variables.tf [hunk 1]
```diff
@@ -34,3 +34,14 @@ variable "project_id" {
 variable "credentials_file_path" {
   description = "Path to service account json"
 }
+
```
Comment (morgante) [inline examples/simple/variables.tf]: @johnrevans6 Looks like some unrelated changes got committed.

File: main.tf [hunk 2]
```diff
@@ -31,8 +44,18 @@ locals {
...
+  Activate services
  *******************************************/
+resource "google_project_service" "activate_services" {
+  count   = "${length(local.services_list)}"
+  project = "${local.project_id}"
+
+  service = "${element(local.services_list, count.index)}"
+}
+
+# /*******************************************
```
Comment (morgante) [inline main.tf]: Why is the `#` added on the beginning of these lines? Please remove.

File: outputs.tf [hunk 3]
```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```
Comment (morgante) [inline outputs.tf]: Please keep this blank line.

File: examples/simple/outputs.tf [hunk 4]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+
```
Comment (morgante) [inline examples/simple/outputs.tf]: Looks like a whitespace error got added. This should be an empty blank line.

File: outputs.tf (hunk 3 above)
Comment (johnrevans6) [reply outputs.tf]: ack
